tests/behat/features/api/deprecated/project/get_featured_programs.feature

```diff
@@ -38,7 +38,7 @@ Feature: Get featured programs
               "ProjectId": "REGEX_STRING_WILDCARD",
               "ProjectName":"Soon to be",
               "Author":"User1",
-              "FeaturedImage": "resources_test/featured/featured_3.jpg"
+              "FeaturedImage": "http://pocketcode.org/resources_test/featured/featured_3.jpg"
```

Comment:
the old API must not change. A new version of an api cannot break an old one. Else the ios app will have major problems